nengo_loihi/tests/test_loihi_api.py

```diff
@@ -22,3 +25,71 @@ def test_overflow_signed(b):
     y, o = overflow_signed(x, bits=b)
     assert np.array_equal(y, z)
     assert np.array_equal(o, q)
+
+
+@pytest.mark.parametrize('offset', (0, 1))
+def test_decay_magnitude(offset, plt, logger):
+    bits = 12
+    # decays = np.arange(1, 2**bits)
+    decays = np.arange(1, 2**bits, 7)
+    ref = []
+    emp = []
+    est = []
+
+    for decay in decays:
+        def empirical_decay_magnitude(decay, x0, invert=False):
+            x = x0 * np.ones(decay.shape, dtype=np.int32)
+            y = x
+            y_sum = np.zeros(decay.shape, dtype=np.int64)
+            for i in range(100000):
+                y_sum += y
+                if (y <= 0).all():
+                    break
+                y = decay_int(y, decay, bits=bits, offset=offset)
+            else:
+                raise RuntimeError("Exceeded max number of iterations")
+
+            return x0 / y_sum if invert else y_sum / x0
+
+        # x0 = np.arange(2**10, 2**11, dtype=np.int32)
+        x0 = np.arange(2**21 - 1000, 2**21, step=41, dtype=np.int32)
+        # x0 = np.arange(2**21 - 1000, 2**21, dtype=np.int32)
```

Comment:
This and two lines above should be removed? Or parametrize the test if we should test with all three.